modules/installation-aws-user-infra-requirements.adoc

```diff
@@ -45,10 +45,7 @@ and approving them.
 * IAM roles
 * S3 buckets
 
-If you are working in a disconnected environment, you are unable to reach the
-public IP addresses for EC2 and ELB endpoints. To resolve this, you must create
-a VPC endpoint and attach it to the subnet that the clusters are using. The
-endpoints should be named as follows:
+If you are working in a disconnected environment or use a proxy, you are unable to reach the public IP addresses for EC2 and ELB endpoints. To resolve this, you must create a VPC endpoint and attach it to the subnet that the clusters are using. Create the following endpoints:
```

Comment:
"are unable to" or "cannot?" 

"to resolve this" or "To reach these IP addresses" (or something like that)